app/javascript/components/search/results/DataRepoStudy.js

```diff
@@ -0,0 +1,66 @@
+/* show a search result entry from Data Repo */
```

Comment:
Starting comments this way gets them picked up in IDEs (e.g., hovering over the module name in an `import`).

```suggestion
/** show a search result entry from Data Repo */
```

Reply:
I clearly need to update my IDE with the correct JS linting!  I think it did it before, but since I upgraded my laptop it's apparently applying the wrong rules.